src/python/py/models/builder.py

```diff
@@ -1177,7 +1240,7 @@ def make_repeat_kv(self, layer_id, root_input, past_kv, present_kv, **kwargs):
         #                Transpose
         #                    |
         #                 Reshape
-        basename = f"/model/layers.{layer_id}/attn/{'k_proj' if past_kv.endswith('key') else 'v_proj'}/repeat_kv"
+        basename = f"/model/layers.{layer_id}/self_attn/{'k_proj' if past_kv.endswith('key') else 'v_proj'}/repeat_kv"
```

Comment:
I think we should revert the change from `attn` to `self_attn`. There will be models in the future that use `cross_attn` and this naming would be wrong for those.

Reply:
Okay, I made this change to make it work with Olive. I think we should do the same change in Olive as well

Reply:
In olive, the names of the nodes are automatically created during torch export which in turn comes from the submodule names from the pytorch model itself. 
I don't think we could always rename self_attn to attn since we don't know what model it might be but maybe we can provide an option to the `convert-adapter` command to do this replacement? https://github.com/microsoft/Olive/blob/main/olive/cli/convert_adapters.py

Reply:
Okay, I will revert this back